integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -70,6 +70,13 @@
                                                     MapGen(StringGen(pattern='key_[0-9]', nullable=False),
                                                            simple_string_to_string_map_gen)]
 
+parquet_datetime_gen_simple = [DateGen(), TimestampGen()]
```

Comment:
Python itself has some issues with dates/timestamps in the far future. Do we need to adjust these so we don't run into issues?  Do we want some tests that explicitly test rebase where we know that the range of dates/timestamps all need to be rebased (or at least it is really likely that they need to be rebased).

Reply:
> Python itself has some issues with dates/timestamps in the far future.

I didn't know that. Of course we can limit the `end` date/timestamp as there will be not any rebase after `1582-10-15` so the test range from `1-1-1` to some day after `1582-10-15` should be good.

Reply:
Some version of python cannot support > 9999 for the year

https://docs.python.org/3/library/datetime.html#datetime.MAXYEAR